webpack/webpack.dev.js

```diff
@@ -2,6 +2,11 @@ const merge = require('webpack-merge');
 const common = require('./webpack.common.js');
 
 module.exports = merge(common, {
+  output: {
+    path: `${__dirname}/dist`,
+    publicPath: '/',
```

Comment:
What is the essence of the other things you added? I think just adding `publicPath` to output should solve this problem. Kindly explain if you have a reason for the filename

Reply:
Yeah, my bad.
I would have taken that off, it was just an oversight.
I would change it now.